internal/controller/children.go

```diff
@@ -167,20 +168,24 @@ func (c ChildResourceUpdates) AllEqual(op controllerutil.OperationResult) bool {
 	return ingressOK && c.Service.UpdateResult == op && c.PVC.UpdateResult == op && c.StatefulSet.UpdateResult == op
 }
 
-func (c ChildResourceUpdates) IsRunning(cr *amaltheadevv1alpha1.AmaltheaSession) bool {
+func (c ChildResourceUpdates) IsRunning(pod *v1.Pod) bool {
 	onlyStatusUpdates := c.AllEqual(controllerutil.OperationResultUpdatedStatusOnly)
 	noUpdates := c.AllEqual(controllerutil.OperationResultNone)
 	ssReady := c.StatefulSet.Manifest.Status.ReadyReplicas == 1 && c.StatefulSet.Manifest.Status.Replicas == 1
```

Comment:
I missed it before but I would suggest to use the k8s short name so it makes parsing easier ( i.e. `stsReady`).